src/main/java/org/ibp/api/java/impl/middleware/common/validator/AttributeValidator.java

```diff
@@ -0,0 +1,42 @@
+package org.ibp.api.java.impl.middleware.common.validator;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.generationcp.middleware.manager.api.GermplasmDataManager;
+import org.generationcp.middleware.pojos.Attribute;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.BindingResult;
+
+import java.util.List;
+
+@Component
+public class AttributeValidator {
+
+
+	@Autowired
+	private GermplasmDataManager germplasmDataManager;
+
+	public void validateAttributeIds(final BindingResult errors, final List<String> attributeIds) {
+		if (attributeIds == null) {
+			return;
+		}
+
+		final List<Integer> ids = (List<Integer>) CollectionUtils.collect(attributeIds, new org.apache.commons.collections.Transformer() {
```

Comment:
@vmaletta this can be done with java8. Please start using functionality that is provided by java8 instead of external libraries when possible

Reply:
@vmaletta Quite sure this validation is wrong for list of attributeIds with duplicated elements. You need to build a set instead of a List to remove duplicated values so that the size comparison works well for such cases

Reply:
Done